### DISCUSSION TITLE
Customize Modals

### DISCUSSION THREAD

**Discussion by 0xCalibur** [rainbow-me/rainbowkit#1501] [Ideas]
I know it's possible to customize ConnectButton but what about the other components? I would like to change the modal fontSize but there doesn't seem to be a way to customize it through the theme

**Comment by DanielSinclair** [comment]
This is not currently supported. Which modal are you looking to customize? Are you using a special font, or why would you like to change the font size?

**Reply by 0xCalibur** [reply]
Oh just wanted to customize the font size because the one I was using  "Press Start 2 Play" is huge compared to normal fonts.
